wercker.yml

```diff
@@ -42,28 +42,37 @@ build:
 
 deploy:
   steps:
+    - add-to-known_hosts:
+        hostname: github.com
+    - add-ssh-key:
+        keyname: GITHUB_KEY
     - script:
-        name: Configure git
+        name: config git
         code: |
           git config --global user.email "devteam@technologyadvice.com"
           git config --global user.name "TechnologyAdvice"
-
-          # remove current .git folder
-          rm -rf .git
     - script:
-        name: gulp build
+        name: checkout branch
+        code: |
+          git checkout $WERCKER_GIT_BRANCH
```

Comment:
Instead of nuking, we just checkout the actual branch being deployed, build it, commit and push it to `gh-pages`.